Animations/vendor/ProvideMake.cmake

```diff
@@ -0,0 +1,51 @@
+# Provides a GNU Make program for use with Unix Makefile's.
```

Comment:
So I know we talked about this, but I have a PR that I'll merge in after this one gets merged that will remove ffmpeg. Because of that we can remove this file since it won't be necessary for the build process anymore :)